README.md

```diff
@@ -15,25 +17,31 @@ package main
 import (
     "fmt"
 
-    "github.com/tebeka/atexit"
+    "github.com/aureolebigben/atexit"
 )
 
 func handler() {
     fmt.Println("Exiting")
 }
 
+func fooHandler() {
```

Comment:
I general I don't like the `foo` and `bar` names. Try to think is a better name - maybe `unregistredHandler`?

Reply:
Think unregistredHandler is a good name, i use fooHandler because I had no inspiration